### PR TITLE
Added cask for nook app

### PR DESCRIPTION
**Pull request by freeslugs**
- tested locally
- audit passed

### CODE REVIEW

File: Casks/nook.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'nook' do
+  version '3.0.0fc9774'
+  sha256 '64f59ccb60394491a3973fd4bae0381be43afcf8e5ea2276cacf299c7e63a88d'
+
+  url 'http://images.barnesandnoble.com/PResources/download/ereader2/bn/mac/NOOKforMac.dmg.zip'
+  name 'NOOK for Mac'
+  homepage 'http://www.barnesandnoble.com/u/nook-for-mac/379003592/'
+  license :gratis
+
+  container :nested => "NOOK for Mac.dmg"
```
Comment (vitorgalvao) [inline Casks/nook.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by freeslugs** [comment]
@vitorgalvao your suggestions have been implemented.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**